src/main/java/org/eclipse/xpanse/terraform/boot/models/plan/TerraformPlan.java

```diff
@@ -17,10 +17,10 @@
 @Builder
 public class TerraformPlan {
 
-    @Schema(description = "Version of terraform")
-    String terraformVersion;
-
     @NotNull
     @Schema(description = "Terraform plan as a JSON string")
     String plan;
+
+    @Schema(description = "Terraform with version")
+    String terraformWithVersion;
```

Comment:
terraformVersionUsed
The version of the Terraform binary used to execute scripts